src/arbitraries/types.ts

```diff
@@ -6,7 +6,7 @@ export type FluentPick<V> = {
 export type ArbitrarySize = {
   value: number
   type: 'exact' | 'estimated'
-  credibleInterval?: [number, number]
+  credibleInterval: [number, number]
```

Comment:
I'm not sure we should force a credible interval for exact sizes... although it may help getting rid of special logic when mixing exact and estimated sizes, like in tuples (which I've marked as a ToDo).

Reply:
I agree with keeping credible intervals in those cases for the sake of having less special code.